api/python/cellxgene_census/tests/test_open.py

```diff
@@ -38,30 +43,54 @@ def test_open_soma_latest() -> None:
         assert isinstance(census, soma.Collection)
 
 
-@pytest.mark.live_corpus
-def test_open_soma_with_context() -> None:
```

Comment:
refactored this existing test into 2 tests, and added a third for the default soma context